packages/angular_devkit/build_ng_packagr/BUILD.bazel

```diff
@@ -56,6 +60,7 @@ pkg_tar(
     tags = ["manual"],
 )
 
+# @external_end
```

Comment:
Nit: Shift this up one line so it looks associated with `pkg_npm()` rather than `ts_library()`.

Reply:
This is actually formatted by buildifier, which we run as part of CI. 
Please see https://github.com/angular/angular-cli/blob/master/package.json#L23

Reply:
ah, it looks like buildifier is still happy after I shifted the comment. All good now.